build.sbt

```diff
@@ -35,7 +35,6 @@ lazy val coverageSettings: Seq[Setting[_]] = {
     "app.*",
     "prod.*",
     "config.*",
```

Comment:
Can you remove the comma from the last element here - the compiler is lenient and ignores this but in some cases this will cause scalastyle to fail the build

Reply:
removed ,